src/hooks/useLinkedAutoComment.ts

```diff
@@ -0,0 +1,67 @@
+import { useCallback, useState } from "react";
+import {
+  useDeskproAppClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { createIssueCommentService } from "../services/space";
+import type { Issue, IssueComment } from "../services/space/types";
+
+export type Result = {
+  isLoading: boolean,
+  addLinkComment: (issue: Issue) => Promise<void|IssueComment>,
+  addUnlinkComment: (issue: Issue) => Promise<void|IssueComment>,
```

Comment:
`Promise<IssueComment>|null` would be easier to manage.

Reply:
fixed